Description: fix type error
 type error Fucntion,it's shuold be Function.
 .
Author: xiao sheng wen <atzlinux@sina.com>

---
Last-Update: 2020-04-19

Index: gmchess/src/ec_throw.cpp
===================================================================
--- gmchess.orig/src/ec_throw.cpp
+++ gmchess/src/ec_throw.cpp
@@ -24,7 +24,7 @@ std::runtime_error throw_except(const ch
 {
 	std::stringstream stream;
 	stream << "File: " 	<< file << std::endl;
-	stream << "Fucntion: " 	<< func << std::endl;
+	stream << "Function: " 	<< func << std::endl;
 	stream << "Expression:"	<< std::endl << line << ":\t" << var << std::endl; 
 	stream	<< "Errno: "	<< errno << std::endl;
 	stream << strerror(errno) << std::endl; 
